drivers/gnss/gnss_quectel_lcx6g.c

```diff
@@ -22,10 +22,8 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(quectel_lcx6g, CONFIG_GNSS_LOG_LEVEL);
 
-#define QUECTEL_LCX6G_STARTUP_DELAY           (K_MSEC(300U))
-#define QUECTEL_LCX6G_STATE_CHANGE_DELAY_MSEC (300LL)
-#define QUECTEL_LCX6G_PAIR_TIMEOUT            (K_SECONDS(11))
-#define QUECTEL_LCX6G_SCRIPT_TIMEOUT_S        (10U)
+#define QUECTEL_LCX6G_PM_TIMEOUT_MS    (500U)
+#define QUECTEL_LCX6G_SCRIPT_TIMEOUT_S (10U)
```

Comment:
hey do you really want to keep these parenthesized? there's no real need for it and they look a bit odd to me

Reply:
what parantheses :D